wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -311,6 +311,19 @@ private extension AppDelegate {
         // Get maxNumberAccounts form SecurityFlags or SessionManager.defaultMaxNumberAccounts if no MAX_NUMBER_ACCOUNTS flag defined
         let maxNumberAccounts = SecurityFlags.maxNumberAccounts.intValue ?? SessionManager.defaultMaxNumberAccounts
 
+        let countlyKey = Bundle.countlyAppKey
+        let host = BackendEnvironment.shared.countlyURL
+
+        let analyticsSessionConfiguration: AnalyticsSessionConfiguration?
+        if let countlyKey, let host {
+            analyticsSessionConfiguration = AnalyticsSessionConfiguration(
+                countlyKey: countlyKey,
+                host: host
+            )
+        } else {
+            analyticsSessionConfiguration = nil
+        }
```

Comment:
I would not see the creation of the analytics session configuration as a responsibility of the AppDelegate. Can it be moved to a `AnalyticsSessionBuilder`?

Reply:
Same here: https://github.com/wireapp/wire-ios/pull/1618#discussion_r1670096616